src/quantum/index-32bit.js

```diff
@@ -106,111 +105,7 @@ export default class QuantumIndex32 extends React.Component {
-                    platform: 'windows7-32-nightly',
-                    option: 'opt',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-amazon-chrome',
-                  },
-                },
-              ]}
-            />,
-            <div style={{ width: '50%' }} />,
-          ],
-          [
-            <PerfherderGraphContainer
-              title='Tp6: Facebook'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    frameworkId: 10,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-facebook-firefox',
-                  },
-                },
-                {
-                  label: 'Chrome',
-                  seriesConfig: {
-                    frameworkId: 10,
-                    platform: 'windows7-32-nightly',
-                    option: 'opt',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-facebook-chrome',
-                  },
-                },
-              ]}
-            />,
-            <PerfherderGraphContainer
-              title='Tp6: Google'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    frameworkId: 10,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-google-firefox',
-                  },
-                },
-                {
-                  label: 'Chrome',
-                  seriesConfig: {
-                    frameworkId: 10,
-                    platform: 'windows7-32-nightly',
-                    option: 'opt',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-google-chrome',
-                  },
-                },
-              ]}
-            />,
-          ],
-          [
-            <PerfherderGraphContainer
-              title='Tp6: YouTube'
-              series={[
-                {
-                  label: 'Firefox',
-                  seriesConfig: {
-                    frameworkId: 10,
-                    platform: 'windows7-32',
-                    option: 'pgo',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-youtube-firefox',
-                  },
-                },
-                {
-                  label: 'Chrome',
-                  seriesConfig: {
-                    frameworkId: 10,
-                    platform: 'windows7-32-nightly',
-                    option: 'opt',
-                    project: 'mozilla-central',
-                    suite: 'raptor-tp6-youtube-chrome',
-                  },
-                },
-              ]}
-            />,
+            null,
```

Comment:
Why `null`?

Reply:
I do not know what else to put as a placeholder: This page demands pairs of charts.

Reply:
Ah OK. I've been using `undefined`.